src/main/java/com/palantir/gradle/versions/VersionsLockPlugin.java

```diff
@@ -450,12 +461,12 @@ private static void ensureNoFailOnVersionConflict(Configuration conf) {
      Since unifiedClasspath depends on the SUBPROJECT_UNIFIED_CONFIGURATION_NAME configuration of all
      subprojects (above), that would resolve them when we resolve unifiedClasspath. We need this workaround
      to enable the workflow:
-
+    
```

Comment:
Filed bug report for this one: https://github.com/palantir/gradle-baseline/pull/800